api/src/main/java/org/pmiops/workbench/workspaces/WorkspaceServiceImpl.java

```diff
@@ -210,8 +191,10 @@ public void deleteWorkspace(DbWorkspace dbWorkspace) {
     // This automatically handles access control to the workspace.
     fireCloudService.deleteWorkspace(
         dbWorkspace.getWorkspaceNamespace(), dbWorkspace.getFirecloudName());
-    dbWorkspace.setWorkspaceActiveStatusEnum(WorkspaceActiveStatus.DELETED);
-    dbWorkspace = workspaceDao.saveWithLastModified(dbWorkspace, userProvider.get());
+    dbWorkspace =
+        workspaceDao.saveWithLastModified(
+            dbWorkspace.setWorkspaceActiveStatusEnum(WorkspaceActiveStatus.DELETED),
```

Comment:
not really related to the other changes

Reply:
why would codacy complain now when previously it did not...?